pythoncalculator/__init__.py

```diff
@@ -1,2 +1,7 @@
 from .add import add 
```

Comment:
I think I found a typo

```suggestion
from .multiply import multiply
```

Reply:
Oops it should be
```suggestion
from .add import add 
from .multiply import multiply
```